src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -173,31 +187,41 @@ private string GetCommitsWorkFolder(IExecutionContext context)
             return commitsRootDirectory;
         }
 
-        private async Task DownloadArtifacts(IExecutionContext executionContext, IList<AgentArtifactDefinition> agentArtifactDefinitions, string artifactsWorkingFolder)
+        private async Task DownloadArtifacts(IExecutionContext executionContext,
+            IList<AgentArtifactDefinition> agentArtifactDefinitions, string artifactsWorkingFolder)
         {
             Trace.Entering();
 
             CreateArtifactsFolder(executionContext, artifactsWorkingFolder);
+
+            executionContext.Output(StringUtil.Loc("RMDownloadingArtifact"));
+
             foreach (AgentArtifactDefinition agentArtifactDefinition in agentArtifactDefinitions)
             {
                 // We don't need to check if its old style artifact anymore. All the build data has been fixed and all the build artifact has Alias now.
                 ArgUtil.NotNullOrEmpty(agentArtifactDefinition.Alias, nameof(agentArtifactDefinition.Alias));
 
                 var extensionManager = HostContext.GetService<IExtensionManager>();
-                IArtifactExtension extension = (extensionManager.GetExtensions<IArtifactExtension>()).FirstOrDefault(x => agentArtifactDefinition.ArtifactType == x.ArtifactType);
+                IArtifactExtension extension =
```

Comment:
breaking into multiple lines is causing more churn... would have been good if you had avoided making this change as part of these changes. is this triggered by VS code editor auto correcting line endings?

Reply:
auto indented by vs.